preprocess_masks/setup_arctic_regions.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+"""
+This script creates a geojson file with Arctic regions
+"""
+from geometric_features import GeometricFeatures, FeatureCollection
+
+regions = {'Baffin Bay': ['Baffin Bay'],
+           'Baltic Sea': ['Skaggerak', 'Kattegat', 'Gulf of Bothnia',
+                          'Gulf of Finland', 'Baltic Sea', 'Gulf of Riga'],
+           'Barents Sea': ['Barents Sea', 'White Sea'],
+           'Canada Basin': ['Canada Basin'],
+           'Canadian Archipelago': ['Canadian Archipelago'],
+           'Central Arctic': ['Central Arctic'],
+           'Chukchi Sea': ['Chukchi Sea'],
+           'East Siberian Sea': ['East Siberian Sea'],
+           'Greenland Sea': ['Greenland Sea'],
+           'Hudson Bay': ['Hudson Bay'],
+           'Irminger Sea': ['Irminger Sea'],
+           'Kara Sea': ['Kara Sea'],
+           'Labrador Sea': ['Labrador Sea'],
+           'Laptev Sea': ['Laptev Sea'],
+           'North Sea': ['North Sea'],
+           'Norwegian Sea': ['Norwegian Sea'],
+           'Beaufort Gyre': ['Beaufort Gyre'],
+           'Beaufort Gyre Shelf': ['Beaufort Gyre Shelf'],
+           'Arctic Ocean (no Barents/Kara Seas)': [
+               'Chukchi Sea', 'Canada Basin', 'East Siberian Sea', 'Laptev Sea',
+               'Central Arctic']}
```

Comment:
@milenaveneziani, can you check this (and really this whole file)?  I think it's right and the resulting geojson looks identical to `arcticRegions.geojson` to me but I want to be sure since some of the constituents are different from what are listed in `arcticRegions.geojson`.  I think that may relate to how you created the arctic regions in `geometric_features`.

Reply:
@xylar: I made these Arctic regions in two steps (there should be a `setup_arcticRegions_partI` and `partII`) because I had to create some intermediate-step regions and change the definition of some other regions, before getting to the final version of 'Arctic regions'. I would have to go back and check, and also test this thoroughly to make sure that we get what we want. Unfortunately, I am on sick leave right now, and will only be able to do a limited amount of work over the next week or two. I would suggest you can take out the Arctic regions from this PR, and we can update the code with them later.